examples/webxr_vr_handinput_profiles.html

```diff
@@ -70,6 +70,8 @@
 				const light = new THREE.DirectionalLight( 0xffffff );
 				light.position.set( 0, 6, 0 );
 				light.castShadow = true;
+				// Prevent shadow acne on the hands
+				light.shadow.bias = -0.0001;
```

Comment:
My solution was to force set `material.side` to `THREE.FrontSide`.: https://github.com/mrdoob/three.js/blob/e9d5ba5f47223cd8f0a4e5c31f2a2990feb4f6a1/examples/jsm/webxr/XRHandMeshModel.js#L26

@toji Any reason the models are double sided?

Reply:
No, there's not. Thanks for pointing that out. I'll fix the meshes and push them to the webxr-input-profiles CDN.

Reply:
Sweet! Let me know when you do that so I can remove the workaround.